ios/brave-ios/Sources/Brave/Frontend/Settings/Features/ShieldsPrivacy/FilterLists/FilterListsView.swift

```diff
@@ -265,6 +346,20 @@ struct FilterListsView: View {
       customRules = nil
     }
   }
+
+  private func updateFilterLists() async {
+    _ = await FilterListStorage.shared.updateFilterLists()
+    await AdblockResourceDownloader.shared.updateResources()
+    await AdBlockGroupsManager.shared.compileEngines()
+  }
+
+  private func updateCustomFilterLists() async {
+    do {
+      try await FilterListCustomURLDownloader.shared.updateFilterLists()
+    } catch {
+      // Handle the error
```

Comment:
Error isn't handled?

Reply:
Yah I should handle this. I put the comment but wasn't sure how to deal with it and forgot about it. Thanks for catching this.

Reply:
Fixed, however since Task.retry swallows errors, the catch will never be triggered. Will fix the issue with Task.retry separately.
![Simulator Screenshot - iPhone 14 Pro - 2024-07-29 at 18 27 52](https://github.com/user-attachments/assets/9e61c5e6-d111-453c-9e24-bbe9754e080d)